samples/bluetooth/broadcast_audio_assistant/src/main.c

```diff
@@ -121,6 +126,84 @@ static bool device_found(struct bt_data *data, void *user_data)
 	}
 }
 
+static bool pa_decode_base(struct bt_data *data, void *user_data)
+{
+	const struct bt_bap_base *base = bt_bap_base_get_base_from_ad(data);
+	uint8_t base_size;
+
+	/* Base is NULL if the data does not contain a valid BASE */
+	if (base == NULL) {
+		return true;
+	}
+
+	base_size = data->data_len - BT_UUID_SIZE_16; /* the BASE comes after the UUID */
+
+	/* Compare BASE and copy if different */
+	if (base_size != received_base_size || memcmp(base, received_base, base_size) != 0) {
+		(void)memcpy(received_base, base, base_size);
+		received_base_size = base_size;
+	}
+
+	/* Stop parsing */
+	k_sem_give(&sem_received_base_subgroups);
+	return false;
+}
+
+static inline bool add_pa_sync_base_subgroup_bis_cb(const struct bt_bap_base_subgroup_bis *bis,
+						    void *user_data)
+{
+	struct bt_bap_bass_subgroup *subgroup_param = user_data;
+
+	subgroup_param->bis_sync |= BIT(bis->index);
+
+	return true;
+}
+
+static void pa_recv(struct bt_le_per_adv_sync *sync,
+			 const struct bt_le_per_adv_sync_recv_info *info,
+			 struct net_buf_simple *buf)
+{
+	bt_data_parse(buf, pa_decode_base, NULL);
+}
+
+static inline bool add_pa_sync_base_subgroup_cb(const struct bt_bap_base_subgroup *subgroup,
```

Comment:
```suggestion
static bool add_pa_sync_base_subgroup_cb(const struct bt_bap_base_subgroup *subgroup,
```

Generally there are rarely any reason to use `inline` for static function.